satellite/satellitedb/storagenodeaccounting.go

```diff
@@ -217,24 +223,39 @@ func (db *StoragenodeAccounting) QueryNodeDailySpaceUsage(ctx context.Context, n
 		err = errs.Combine(err, rows.Close())
 	}()
 
-	var nodeSpaceUsages []accounting.NodeSpaceUsage
+	var nodeStorageUsages []accounting.NodeStorageUsage
 	for rows.Next() {
 		var atRestTotal float64
 		var startTime time.Time
+		var prevAtRestTotal sql.NullFloat64
+		var prevStartTime dbutil.NullTime
 
-		err = rows.Scan(atRestTotal, startTime)
+		err = rows.Scan(&atRestTotal, &startTime, &prevAtRestTotal, &prevStartTime)
 		if err != nil {
 			return nil, Error.Wrap(err)
 		}
 
-		nodeSpaceUsages = append(nodeSpaceUsages, accounting.NodeSpaceUsage{
+		// skip first entry as we can not extract hours
+		// properly without storagenode storage tallies
+		// which formed this value
+		if !prevStartTime.Valid {
+			continue
+		}
+
+		atRest := atRestTotal - prevAtRestTotal.Float64
```

Comment:
Why not calculate this directly in SQL query?

Reply:
Because I am not sure how do a query that will work with both `postgres` and `sqlite`

Reply:
Are there separate queries that we can write?